client/client_test.go

```diff
@@ -193,32 +195,52 @@ func TestAddLeaf(t *testing.T) {
 	cli := trillian.NewTrillianLogClient(env.ClientConn)
 	for _, test := range []struct {
 		desc    string
+		leaf    []byte
 		client  trillian.TrillianLogClient
 		wantErr bool
```

Comment:
QueueLeaf doesn't really have any if statements inside of it to test.  I wonder if I should just remove this test function?